index.html

```diff
@@ -0,0 +1,64 @@
+<html>
+<head>
+  <title>The Lorem Micro Blog</title>
```

Comment:
I missed this on mine.  but I'm wondering if the title intended in the original mockup was "A Web Page"